### DISCUSSION TITLE
Is it possible to return only the created nested records from an update call?

### DISCUSSION THREAD

**Discussion by gjritter** [prisma/prisma#2367] [Q&A] [answered]
When I make an update call that is creating nested objects, like this:

```
prisma.store.update({
    where: {
        storeId
    },
    data: {
        product: {
            create: {
                // properties of the new product
            }
        }
    },
    select: {
        product: true
    }
});
```

the response contains all of the products for the store, instead of only the newly created one.

Is there some mechanism to tell Prisma to only select the nested objects that were created by the nested creates within the update?

**Comment by ryands17** [comment, marked as answer]
Hey @gjritter :wave: 
Currently with this approach it wouldn't be possible just to fetch the newly created records, so as a workaround, you would have to fetch it again using the created post id's in a separate query.

Please open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this further :)

**Comment by ludralph** [comment]
Hi @gjritter,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
